### ISSUE TITLE
NVDA becomes unresponsive when closing windows 10 mail

### ISSUE THREAD

**Issue by nidza07** [nvaccess/nvda#7911]
### Steps to reproduce:
 1. Open Windows 10 mail and read some messages.
2. Close it by pressing alt plus f4.
### Expected behavior:
 NVDA should behave normally like it does when closing any other app.

### Actual behavior:
 NVDA fails to read any focus changes, and only NVDA specific commands like NVDA plus up arrow or NVDA plus tab work.

### System configuration: 
Dell Laptop with 4 gigs of RAM, and a dual core 3.0 GHZ processor.
NVDA version:
 next-14783,eda951dd

NVDA Installed or portable:
Installed
Other information:
 Log is attached

[NVDA log.txt](https://github.com/nvaccess/nvda/files/1636235/NVDA.log.txt)
Windows version:
 : Windows 10 64-bit Version 1609 , build 16299

Name and version of other software in use when reproducing the issue:
Windows 10 mail, version 17.8730.21725.0
### Other questions:

Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
> No

**Comment by josephsl** [comment]
Hi, how about restarting with add-ons disabled? Thanks.

**Comment by nidza07** [comment]
Hello. The issue does not happen every time I do the STR mentioned so it is a bit harder to reproduce each time. However, I just tried with addons disabled and it still does happen.

**Comment by josephsl** [comment]
Hmmm, I see. I suggest filing a feedback with MS so we can upvote it. CC @MichaelDCurran

**Comment by Adriani90** [comment]
@nidza07 could you please test with last NVDA Alpha if this issue is still occuring? The last Alpha version is here:
https://ci.appveyor.com/project/nvaccess/nvda/build/alpha-16982,01118965

Thanks!

**Comment by Adriani90** [comment]
I am testing with NVDA 2019.1, the issue is not occuring anymore. Closing as works for me. Please comment on it if you stillhave this issue and we can reopen it. Thanks!

**Comment by josephsl** [comment]
Hi,

Caused by same problem as #9341 - COM error exception not handled, actually resolved in 2019.2, not 2019.1. The reason why people won't come across this in 2019.1 is due to Windows 10 App Essentials add-on shipping with the fix for this bug.

Thanks.

**Comment by Adriani90** [comment]
Reopening until it is solved by a merged PR.

**Comment by lukaszgo1** [comment]
It is solved now #9392 is in master already.

**Comment by Adriani90** [comment]
Closing.
